doc/api/errors.md

```diff
@@ -2391,6 +2381,16 @@ Used when an invalid character is found in an HTTP response status message
 -->
 A given index was out of the accepted range (e.g. negative offsets).
 
+<a id="ERR_INVALID_OPT_VALUE"></a>
+### `ERR_INVALID_OPT_VALUE`
```

Comment:
For legacy errors, I think we usually add YAML metadata indicating when an error was added and removed. But I don't know how/when it is added.

Reply:
(This can probably land without it and a follow-on PR can add it, but if anyone knows what kind of YAML placeholders to add here, let's get 'em added.)

Reply:
Found: https://github.com/nodejs/node/pull/11317 and https://github.com/nodejs/node/pull/11300, those will be v9 and v8 respectively.
I'll add them to also re-run the GitHub Actions run.